scripts/globals/abilities/animated_flourish.lua

```diff
@@ -3,55 +3,52 @@
 -- Provokes the target. Requires at least one, but uses two Finishing Moves.
 -- Obtained: Dancer Level 20
 -- Finishing Moves Used: 1-2
--- Recast Time: 00:30
+-- Recast Time: 00:00:30
 -----------------------------------
-require("scripts/globals/settings")
 require("scripts/globals/status")
 require("scripts/globals/msg")
 -----------------------------------
 
 function onAbilityCheck(player,target,ability)
-
-    if (player:hasStatusEffect(tpz.effect.FINISHING_MOVE_1)) then
-        return 0,0
-
-    elseif (player:hasStatusEffect(tpz.effect.FINISHING_MOVE_2)) then
-        return 0,0
-
-    elseif (player:hasStatusEffect(tpz.effect.FINISHING_MOVE_3)) then
-        return 0,0
-
-    elseif (player:hasStatusEffect(tpz.effect.FINISHING_MOVE_4)) then
-        return 0,0
-
-    elseif (player:hasStatusEffect(tpz.effect.FINISHING_MOVE_5)) then
+    if
+        player:hasStatusEffect(tpz.effect.FINISHING_MOVE_1) or
+        player:hasStatusEffect(tpz.effect.FINISHING_MOVE_2) or
+        player:hasStatusEffect(tpz.effect.FINISHING_MOVE_3) or
+        player:hasStatusEffect(tpz.effect.FINISHING_MOVE_4) or
+        player:hasStatusEffect(tpz.effect.FINISHING_MOVE_5)
+    then
```

Comment:
This logic comparison forces 5 checks every time. The old only makes checks until it finds a true statement, somewhere between 1 and 5 checks.